### PR TITLE
Changes in AMD64 connectedk8s cli for ARM64 clusters

### PR DESCRIPTION
**Pull request by atharvamulmuley**
### PR Description
* Removed ARM64 unsupported warning
* Increased helm delete timeout for ARM64 clusters
* Added multi-architectural images for troubleshoot
* Delete azure-arc-release NS if exists as part of delete command

### CODE REVIEW

**Comment by yonzhan** [comment]
connectedk8s

File: src/connectedk8s/azext_connectedk8s/custom.py [hunk 1]
```diff
@@ -719,6 +731,12 @@ def delete_connectedk8s(cmd, client, resource_group_name, cluster_name,
...
     release_namespace = get_release_namespace(kube_config, kube_context, helm_client_location)
 
+    utils.try_list_node_fix()
+    api_instance = kube_client.CoreV1Api()
+    node_api_response = utils.validate_node_api_response(api_instance, None)
+    is_arm64_cluster = check_arm64_node(node_api_response)
+    
+
     # Check forced delete flag
     if(force_delete):
```
Comment (sirireddy12) [inline src/connectedk8s/azext_connectedk8s/custom.py:741]: did we test force delete as well on arm64? current timeout is 20s for CRD deletion command, would be good to confirm if that works for arm64 too

Comment (atharvamulmuley) [reply src/connectedk8s/azext_connectedk8s/custom.py:726]: Yeah I tested that, it is working with the current changes

**Review by sirireddy12 — APPROVED** [review]

**Comment by zhoxing-ms** [comment]
Could you add some tests for this PR?

**Comment by atharvamulmuley** [comment]
> Could you add some tests for this PR?

@zhoxing-ms we are adding tests for all the code as part of this PR: https://github.com/Azure/azure-cli-extensions/pull/5566

**Review by zhoxing-ms — APPROVED** [review]

**Comment by zhoxing-ms** [comment]
@atharvamulmuley Could you please add some description for this PR in the `HISTORY.rst`? In addition, if you want to release a new version to release this extension, please also upgrade the version defined in the `setup.py`

**Comment by atharvamulmuley** [comment]
> @atharvamulmuley Could you please add some description for this PR in the `HISTORY.rst`?

Added the description in the PR. History and version bump will happen in the next PR that will be merged after this.

**Pull request merged**